winch/codegen/src/codegen/control.rs

```diff
@@ -92,9 +144,30 @@ impl ControlStackFrame {
     ) -> Self {
         let result = <M::ABI as ABI>::result(&returns, &CallingConvention::Default);
         let mut control = Self::Block {
-            original_stack_size: 0,
+            original_stack_len: 0,
             result,
+            branch_target: false,
```

Comment:
Can we call this field something more like `exit_reachable`? `branch_target` makes me think this is a label; really this is a reachability flag for some other program point than the current one.

Reply:
Actually, after reading further: I was struggling to understand how this got set when the block "falls through"; it turns out it it isn't, and rather that case is implicitly handled because if we reach an `End` for a block while in reachable code, we'll remain reachable. So maybe something like `is_branch_target` makes it clear it's a predicate.

Reply:
Updated!